docs/source/v2/central-app.blade.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Central App
+description: Central App
+extends: _layouts.documentation
+section: content
+---
+
+# Central App {#central-app}
+
+This package uses routes to separate the tenant part of the application from the central part of the application. The central part will commonly include a landing page, sign up form, and some sort of dashboard.
+
+## Central routes {#central-routes}
+
+Routes in the `routes/web.php` file are the central routes. When they are visited, tenancy is *not* intialized and any model, cache call, controller, job dispatch, Redis call and anything else that is called in during this request will be central.
+
+## Central domains {#central-domains}
+
+However, since you don't want routes related to the app on your main domain and sign up forms on tenant domains, you must also define what domains host the central stuff in the `tenancy.exempt_domains` config.
+
+## Using central things inside the tenant app {#using-central-things-inside-the-tenant-app}
+
+To use central things (databases/caches/Redis connections/filesystems/...) on special places of your tenant app, you may do the following.
+
+## Central database {#central-database}
```

Comment:
I wonder if this and the headings following it should use `###` and be treated as sub-headings of the one above?

Reply:
Good catch.